supertokens_python/recipe/emailpassword/emaildelivery/service/smtp/implementation.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
```

Comment:
create a folder called service_implementation and move implementation.py & email_verification_implementation.py  to that folder. implementation.py will become __init__.py in the new folder